flutter/pubspec.yaml

```diff
@@ -27,6 +24,12 @@ dev_dependencies:
   yaml: ^2.2.1 # needed for version match (code and pubspec)
   pedantic: ^1.9.2
 
+dependency_overrides:
+  sentry:
+    # TODO: we need the specific version for publishing it, but the relative path for development.
+    # Figure out how to do it via Craft for releases.
+    path: ../dart
```

Comment:
```suggestion
dependency_overrides:
  sentry:
    path: ../dart
```

Reply:
looks like this solves the TODO

Reply:
but CI fails: `Because every version of sentry_flutter from path depends on sentry >=4.0.0-beta.2 <4.1.0 which doesn't match any versions, sentry_flutter from path is forbidden.`

Reply:
ok to solve this we need to apply this https://github.com/getsentry/sentry-dart/pull/229/files#r537115550

https://dart.dev/tools/pub/dependencies#dependency-overrides
there's a warning note about it.

> Using a dependency override involves some risk. For example, using an override to specify a version outside the range that the package claims to support

I think it's fine, we are bumping the version manually for now, and later we'll automatize this.

Reply:
This is actually just the example failing, I think I can fix this.

Reply:
oh I see, good catch, thanks